modules/opsgenie-team/versions.tf

```
@@ -0,0 +1,14 @@
+terraform {
+  required_version = ">= 1.0.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.0"
```

Are we ready to pin any components past 3.x?

```suggestion
      version = "~> 3.0"
```

---

Yeah only our s3 component really prevented us

---

meaning this can be pinned to 4.0